agentmanigest.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: v1
```

Comment:
The filename has a typo: -> `agentmanifest.yaml`.

What are these files for?

Reply:
Ooops, `agentmanigest.yaml` and `agentconfig.yaml` should not be here. They are part of a manual test I was doing and I added them by mistake. Going to delete them right now.